### PR TITLE
Allow anonymous binds for LdapUserInfo

### PR DESCRIPTION
**Pull request by jasonhancock**

### CODE REVIEW

**Comment by djmitche** [comment]
looks good to me

File: master/buildbot/www/ldapuserinfo.py [hunk 1]
```diff
@@ -59,9 +59,14 @@ def connectLdap(self):
         # define the server and the connection
         s = ldap3.Server(netloc[0], port=int(netloc[1]), use_ssl=server.scheme == 'ldaps',
                          get_info=ldap3.GET_ALL_INFO)
+
+        auth = ldap3.AUTH_SIMPLE
+        if (self.bindUser is None and self.bindPw is None):
```
Comment (sa2ajj) [inline master/buildbot/www/ldapuserinfo.py]: the `(` and `)` are not needed here.

**Comment by sa2ajj** [comment]
sorry for nitpicking

**Comment by jasonhancock** [comment]
Parenthesis removed and commits squashed.

**Comment by tardyp** [comment]
thank you

**Pull request merged**